### PR TITLE
updates to KSPM FAQ

### PR DESCRIPTION
**Pull request by tinnytintin10**
Updated KSPM FAQ section to include some more info about supported k8s version

Preview: [KSPM FAQ](https://security-docs_2707.docs-preview.app.elstc.co/guide/en/security/master/kspm-faq.html)

### CODE REVIEW

**Review by oren-zohar — APPROVED** [review]

File: docs/cloud-native-security/kspm-faq.asciidoc [hunk 1]
```diff
@@ -3,7 +3,13 @@
...
 
 [discrete]
-== Clusters
+== Kubernetes Clusters
+
+*What versions of Kubernetes are supported?*
+
+For self-managed / vanilla clusters, Kubernetes version 1.23 is supported. 
+
+For EKS clusters, all available Kubernetes versions at the time of cluster deployment are supported.
```
Comment (benironside) [inline docs/cloud-native-security/kspm-faq.asciidoc:12]:
```suggestion
For EKS clusters, all Kubernetes versions available at the time of cluster deployment are supported. 
```

**Review by benironside — COMMENTED** [review]
Looks good, thanks, I'll get this merged

File: docs/cloud-native-security/kspm-faq.asciidoc [hunk 2]
```diff
@@ -3,7 +3,13 @@
 Frequently asked questions about the Kubernetes Security Posture Management (KSPM) integration and features.
 
 [discrete]
-== Clusters
+== Kubernetes Clusters
+
+*What versions of Kubernetes are supported?*
+
+For self-managed / vanilla clusters, Kubernetes version 1.23 is supported.
```
Comment (nastasha-solomon) [inline docs/cloud-native-security/kspm-faq.asciidoc:10]:
```suggestion
For self-managed/vanilla clusters, Kubernetes version 1.23 is supported. 
```

**Review by nastasha-solomon — APPROVED** [review]
Left one optional edit. Otherwise, LGTM!

**Pull request merged**